pyramid-app/{{ cookiecutter.slug }}/tests/unit/conftest.py

```diff
@@ -0,0 +1,24 @@
+import pytest
```

Comment:
Again, this `conftest.py` file will be created in newly-created projects but `make template` won't create or update it in existing projects.